README.md

```diff
@@ -30,6 +30,18 @@ It will listen on <http://127.0.0.1:3030> by default.
 Note: If you have issues with `npm install` please update to npm 2.0+ using `npm install -g npm@2` 
 ([Issue #1594](https://github.com/mozilla/fxa-content-server/issues/1594))
 
+## Docker Based Development
+
+To run the content server via Docker, three steps are required:
+
+    $ docker build --rm -t mozilla/fxa_content_server
```

Comment:
When I ran this command locally, from the fxa-content-server directory, I had to append a `.`. This is on OSX using boot2docker/docker 1.5.0